src/types.ts

```diff
@@ -2100,6 +2101,7 @@ export type ParameterWithValueList = ParameterWithValue[] | ParameterWithValueLi
 interface EVMTokenCreateParamsDto {
     contractId: string;
     deployFunctionParams?: Array<ParameterWithValue>;
+    useGasless?: boolean;
```

Comment:
the 'useGasless' should be in IssueTokenRequest and not here I believe

Reply:
changed